index.test.js

```diff
@@ -0,0 +1,48 @@
+import { add, addString, toNumber, filterOrange } from './index';
+
+test('add 1 + 2 to equal 3', () => {
+	expect(add(undefined, 2)).toBe(NaN);
+});
+
+//Task01
+/*
+・aとbが"a"、"b"の時"ab"となる
+・aが"a", bが1の時"a1"が返る
+・aが1、bが1の時-1が返る
+*/
+
+describe('addString', () => {
+	test('add a + b to match ab', () => {
+		expect(addString('a', 'b')).toMatch('ab');
+	});
+
+	test('add a + 1 to match a1', () => {
+		expect(addString('a', 1)).toMatch('a1');
+	});
+
+	// if = false else return -1
```

Comment:
this comment no need
it you should write to describe function as first argument.

Reply:
I'm sorry I forgot to delete the comment.
I deleted the comment.

[8ac52e7](https://github.com/nyattsu72/handson-jest/pull/1/commits/8ac52e7dc0a88abf57612ac6dbbc2de89316ef00)